docs/creating_component_deps_and_reqs.md

```diff
@@ -22,6 +22,23 @@ Requirements is a list of strings. Each entry is a `pip` compatible string. For
 REQUIREMENTS = ['pychromecast==0.6.12']
 ```
 
+### Note
+
+Be aware that actual python imports of these dependencies should be done inside functions that use them. Without it you will get errors on the top-level imports.
```

Comment:
Replace "Without…" with: 

> This is because Home Assistant installs requirements on demand and so the requirement won't be loaded the first time your component is loaded.

Reply:
done.